src/components/common/Actions.vue

```diff
@@ -149,6 +144,43 @@ export default class Actions extends Mixins(FilingTemplateMixin, LegalApiMixin)
     }
     this.setIsFilingPaying(false)
   }
+
+  private get nextRoute (): string | null {
+    const nextStep = this.next()
+    if (nextStep) {
+      return nextStep.to
+    }
+    return null
+  }
+
+  private get nextButtonLabel (): string {
+    const nextStep = this.next()
+    if (nextStep) {
+      return nextStep.text.replace('\n', ' ')
+    }
+    return ''
+  }
+
+  private next () : any {
+    const currentStep: number| null = this.$router.currentRoute.meta
+      ? this.$router.currentRoute.meta.step : null
+    if (currentStep && currentStep < this.stateModel.maxStep) {
```

Comment:
Maybe applies here as well. `currentStep? < this.stateModel.MaxStep`. 

No real action required here, just purely suggestion if you feel optional chaining is a feature we should implement more to do away with short-circuiting properties.